.github/workflows/db.yaml

```diff
@@ -106,6 +106,7 @@ jobs:
       - name: Install dependencies
         run: |
           remotes::install_deps(dependencies = TRUE)
+          remotes::install_cran('pkgbuild')
```

Comment:
I think it would be better to switch to `uses: r-lib/actions/setup-r-dependencies@v2` rather than adding more  exceptions.

Reply:
Done / migrated to using setup-r-dependencies.

I am not entirely convinced the package cache ( in this step / setup-r-dependencies ) is used in any way;  I can chase that down later, if that works for you.